selfdrive/car/hyundai/values.py

```diff
@@ -2052,7 +2052,7 @@ def match_fw_to_car_fuzzy(live_fw_versions) -> Set[str]:
 
 # these cars have not been verified to work with longitudinal yet - radar disable, sending correct messages, etc.
 UNSUPPORTED_LONGITUDINAL_CAR = LEGACY_SAFETY_MODE_CAR | {CAR.KIA_NIRO_PHEV, CAR.KIA_SORENTO, CAR.SONATA_LF, CAR.KIA_OPTIMA_G4_FL,
-                                                         CAR.KIA_OPTIMA_H_G4_FL}
+                                                         CAR.KIA_OPTIMA_H_G4_FL, CAR.KONA_EV_2ND_GEN, CAR.IONIQ_6}
```

Comment:
Unsupported longitudinal is for cars that we have no intention of enabling because either they do not work (radar disable fails), or not verified to work at all

Reply:
In this case, radar disable fails every time the users have tested for these 2 platforms on HDA2. Would that suffice the requirement to have them on this list?

Reply:
We initially tried disabling the ADAS Driving ECU and failed every time (SCC messages are still being broadcasted), we then moved onto disabling only the radar ECU and ADAS Driving ECU + radar ECU - all attempts have failed to disable.

Reply:
Ah I see, if all attempts to silence both ECUs do not work then this is fine. I thought you were using the list to show experimental long